solr/core/ivy.xml

```diff
@@ -133,6 +133,7 @@
     <dependency org="org.codehaus.janino" name="janino" rev="${/org.codehaus.janino/janino}" conf="compile"/>
     <dependency org="org.codehaus.janino" name="commons-compiler" rev="${/org.codehaus.janino/commons-compiler}" conf="compile"/>
     <dependency org="com.google.protobuf" name="protobuf-java" rev="${/com.google.protobuf/protobuf-java}" conf="compile"/>
+    <dependency org="com.jayway.jsonpath" name="json-path" rev="${/com.jayway.jsonpath/json-path}" conf="compile"/>
```

Comment:
Needed for Calcite upgrade

Reply:
Need to address precommit failures from this. Check license/notice. Fixed initial test failures about class not found though. Will get to later tonight.

Reply:
Addressed the license/notice.